problems/arraysIntro.js

```diff
@@ -1,60 +1,84 @@
 /**
  * Appends a fruit to an array of fruits and returns the array.
- * @param {string[]} fruits
- * @param {string} fruit
+ * @param {string[]} fruits;
+ * @param {string} fruit;
  * @returns {string[]}
  */
-function addFruitToEnd() {}
+function addFruitToEnd(fruits, fruit) {
+  fruits.push(fruit);
+    return fruits;
+
+}
 
 /**
  * Adds a fruit to the front of an array of fruits and returns the array.
  * @param {string[]} fruits
  * @param {string} fruit
  * @returns {string[]}
  */
-function addFruitToFront() {}
+function addFruitToFront(fruits, fruit) {
+fruits.unshift(fruit);
```

Comment:
Be sure to tab everything in the functions code block to maintain proper formatting and to make sure it's easily readable.

Reply:
ok will do.